src/main/java/org/spongepowered/api/text/Texts.java

```diff
@@ -435,6 +435,29 @@ public static String toPlain(Text text, Locale locale) {
     public static String toJson(Text text, Locale locale) {
         return factory.toJson(text, locale);
     }
+
+    /**
+     * Parses the provided XML-formatted message to a Text object.
+     *
+     * @param input The input string to parse
+     * @return The parsed {@link Text}
+     */
+    public static Text parseXml(String input) {
+        return factory.parseXml(input);
+    }
+
+    /**
+     * Returns an xml representation of the {@link Text} for usage in configurations.
+     * TODO: Does Locale make sense here?
```

Comment:
Looking at some of the other methods, `Locale` is an overload parameter
(e.g `toJson(Text)` and `toJson(Text, Locale)` so perhaps provide a `toXml(Text)` method too